samples/v1.0/Scenarios/WeatherLarge.json

```diff
@@ -74,18 +79,15 @@
 							"altText": "Drizzly weather"
 						},
 						{
-							"type": "FactSet",
-							"horizontalAlignment": "right",
-							"facts": [
-								{
-									"title": "High",
-									"value": "50"
-								},
-								{
-									"title": "Low",
-									"value": "32"
-								}
-							]
+							"type": "TextBlock",
```

Comment:
can you add horizontal-alignment-right?
the new text looks off-centered.

Reply:
elements have been centered now